python/tvm/relay/frontend/onnx.py

```diff
@@ -195,6 +195,17 @@ def _dim_check(attrs):
     return _dim_check, "Only 1d, 2d and 3d kernel supported."
 
 
+def get_scalar(x, params, dtype="float32"):
+    """Helper to get a scalar value for Quantized operators."""
+    if isinstance(x, _expr.Var) and x.name_hint in params:
```

Comment:
Should we throw an error if it's a var we can't look up in params

Reply:
QLinearConv depends on falling back to `op.cast` in the case that x.name_hint is not in params, not throwing error.